catroid/src/main/java/org/catrobat/catroid/utils/CrashReporter.java

```diff
@@ -59,12 +60,24 @@ public static boolean initialize(Context context) {
 					.build();
 			Fabric.with(context, crashlyticsKit);
 			Log.d(TAG, "INITIALIZED!");
+			setIsInitialized(true);
 			return true;
 		}
 		Log.d(TAG, "INITIALIZATION FAILED! [ Report : " + isReportingEnabled() + "]");
+		setIsInitialized(false);
 		return false;
 	}
 
+	@VisibleForTesting
+	public static void setIsInitialized(boolean initialized) {
+		isInitialized = initialized;
+	}
+
+	@VisibleForTesting
+	public static boolean getIsInitialized() {
+		return isInitialized;
+	}
```

Comment:
A boolean getter is normally called `isX` or `hasX`.
Method names like `isInitialized` and `setInitialized` are better imo.